config/default/block_content.type.uiowa_articles.yml

```diff
@@ -1,9 +1,8 @@
 uuid: cbea77a6-0e50-488a-8d6c-c228b11dc1b9
 langcode: en
 status: true
-dependencies:
-  - sitenow_articles
+dependencies: {  }
```

Comment:
Is this an intentional change?

Reply:
i think it was an automatic thing.

Reply:
I'm not sure this matters. If the dependency exists in code but isn't recognizable via config, we could change it to use `enforced`: https://www.drupal.org/node/2404447